tests/test_api_v1.py

```diff
@@ -1080,10 +1071,18 @@ def test_set_state_stale(
     db.session.add(request)
     db.session.commit()
 
+    bundle_dir = RequestBundleDir(1, str(tmpdir))
+    bundle_dir.mkdir(parents=True)
```

Comment:
@tkdchen Could you please explain where the test actually remove the directory? 
and why did we use mocks for that before? :)

Reply:
@akhmanova 

For the first question, https://github.com/release-engineering/cachito/blob/master/cachito/web/api_v1.py#L494

For the second one, I don't know why :)